### ISSUE TITLE
[Hero] Invokers sister

### ISSUE THREAD

**Issue by zelding** [AngelArenaAllstars/suggestions#401]
Lore:
-----

Invokers modest sister. 'Nuff.

Mechanix:
---------

the first cast determines the effect of the spell (_select_),
the second the form (_cast_); so order matters above all.

The spell is _cast_ when the second skill was cast, no targeting required.

Her ult shoudl be considered a regular spell (like Udyr in the unnamed game)

After _selecting_ a spell you should have several seconds to chose a form (_cast_)

The _selected_ ability should be disabled until you _cast_ it, so QQ/WW/EE/RR combinations are not allowed.
I've tried to create the skill orders in the examples in a way that will not result in strictly better spells.
Your damaging spells are: a skillshot (RQ), a close-target self damaging spell (RW) and a dash-attack (RE); I've made sure that you cant do the large aoe dmg - 'cuz thats not interesting. 
With the same logic you can't do: debuffs in a line, heal yourself (and the nearest ally), buff allies after the dash.

On second thought it might be an okay Aghanim's upgrade, to allow duplicate skillcasts  such as QQ.

Using the below examples:
 - if you cast `R` then `Q` you will fire a "missile" ahead and behind damaging enemies
 - if you cast `E` then `W` you will buff yourself and the nearest ally (like Sona `W` in the unnamed game)

**effect examples (first cast -** _select_**):**

Q: debuff  | affects enemies | armor, mspd, stat, healing, disarm, break, mute
W: heal    | affects allies  |
E: buff    | affects allies  | armor, mspd, stat, healing, aspd, mrez
R: damage  | affects enemies | 

**forms examples (second cast -** _cast_**):**

Q: forward and backward projectile                         | magic damage
W: nearest ally+self / enemy                               | pure damage
E: dash forward and "attack" in a short wide cone in fornt | physical  damage
R: circle aoe, self center                                 | magic damage

**Comment by Wujekklawy** [comment]
Neat idea but have same problem as invoker had, a lot of this feels kike a filler spells

**Comment by Wujekklawy** [comment]
Neat idea but have same problem as invoker had, a lot of this feels like a filler spells wigh not mutch of a synergy with itself

**Comment by zelding** [comment]
Not necessarily, all the spells I've listed are just examples to demonstrate the mechanics behind her design.

If *just* do the math, than yeah I agree, it sounds like she has 12 spells; but if you try my perspective, than she only has 4 spells, that have 3 different forms.

In my mind she should be at most a bit above average if you try to fit her to a role, but she should fit nicely to any.

**Comment by Wujekklawy** [comment]
Also whats the difrence between R R and R W? Aside that one is pure and other one not? Also is * Q a skillshot? Also Q * and E * aplly all of those de/buffs? Aside of this they are basicaly same abbility but one for allays and one for enemies

**Comment by Wujekklawy** [comment]
16, not 12

**Comment by Wujekklawy** [comment]
Also there is one thing, don't try to do hero that can do evrything, you'll end up with one that can do nothing i also tried do that but end up scraching it

**Comment by zelding** [comment]
It's 12 skills only, you can't do QQ or any doubles;

Q and E shouldn't apply all, I just listed some possible effects.

Yes, none of her spells should require explicit targeting.

**Comment by Wujekklawy** [comment]
Concider yourself lucky, noone cared to read and pin out flaws in mine ones ;D

**Comment by Wujekklawy** [comment]
Anyway, i advice to ad a disclamer that cant double tap for spels

**Comment by Wujekklawy** [comment]
If youre willing to read mine ones i suggest to start with [ReadMe] i added there pretty important things as well as list of them

**Comment by zelding** [comment]
That was a good idea, I've edited the post

**Comment by Wujekklawy** [comment]
One little sligthly kinda probably one of mayby most important things, **TALENT TREE**
